users/views.py

```diff
@@ -0,0 +1,26 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from .models import User
+from .serializers import UserSerializer
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+class UserDetail(APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+    def get(self,request,pk):
+        user = get_object_or_404(User,pk=pk) 
+        serializers = UserSerializer(user)
+        return Response(serializers.data)
```

Comment:
don't forget to return the HTTP status code with the response same for the rest of endpoint